src/77480.c

```diff
@@ -783,7 +787,59 @@ void render_player(void) {
     }
 }
 
-INCLUDE_ASM(void, "77480", render_player_model);
+void render_player_model(void) {
+    RenderTask task;
+    RenderTask* rtPtr = &task;
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    s32 x, y, z;
+    s8 renderModeTemp;
+    void (*phi_a0)(void);
```

Comment:
Type this as `void (*appendGfx)(void*)` and you can get rid of the `void*` cast on L836

Reply:
May also need to make `appendGfx_player` take an unused void* arg